pkg/gce-cloud-provider/compute/fake-gce.go

```diff
@@ -260,45 +260,32 @@ func (cloud *FakeCloudProvider) InsertDisk(ctx context.Context, volKey *meta.Key
 		}
 	}
 
-	var diskToCreate *CloudDisk
+	computeDisk := &computev1.Disk{
+		Name:             volKey.Name,
+		Zone:             volKey.Zone,
```

Comment:
It looks like computeDisk.Zone is set in the meta.Zonal case below, and unused in the meta.Regional case. Maybe best not to set it here?

Reply:
You are right. Removed